hypha/apply/api/v1/views.py

```diff
@@ -72,6 +72,8 @@ def filter_unique_options(self, options):
         return list(filter(lambda x: len(x.get("label")), unique_items))
 
     def format(self, filterKey, label, options):
+        if label == "Screenings":
+            options.insert(0, {"key": None, "label": "No Screening"})
```

Comment:
I want the no screening option to go first. Is this the best way to do it?

Reply:
Yeah, looks good.